### DISCUSSION TITLE
Any plan support dameng database?

### DISCUSSION THREAD

**Discussion by baoyachi** [diesel-rs/diesel#3268] [Ideas]
ref link: https://www.dameng.com/view_61.html

**Comment by weiznich** [comment]
We are not interested in adding support for more database backends to diesel itself as the crate is already large. That written: Diesel is designed in such a way that third party crates can provide custom connection and backend implementations. Checkout the relevant documentation [here](https://docs.diesel.rs/master/diesel/connection/trait.Connection.html#implementing-a-custom-connection) for details. With [diesel-oci](https://github.com/GiGainfosystems/diesel-oci) and [rsfbclient](https://github.com/fernandobatels/rsfbclient) there are already existing crates using this approach that can be used as inspiration on how to implement such a backend.
